st2common/st2common/rbac/types.py

```diff
@@ -184,7 +184,8 @@ def get_resource_type(cls, permission_type):
             return ResourceType.EXECUTION
 
         split = permission_type.split('_')
-        assert len(split) >= 2
+        if not len(split) >= 2:
```

Comment:
Weird comparison here.

```suggestion
        if len(split) < 2:
```